### PR TITLE
docs: Update keyConcepts

### PR DESCRIPTION
**Pull request by YuriKravetc**
Fixing some typos

### CODE REVIEW

File: docs/keyConcepts.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ sort: 22
 
 ## Data model
 
-### What is a metric
+### What is a metric?
```
Comment (hagen1778) [inline docs/keyConcepts.md:9]: no need in question mark here

File: docs/keyConcepts.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ sort: 22
...
 
-Simply put, `metric` is a numeric measure or observation of something.
+Simply put, the `metric` is a numeric measure or observation of something.
```
Comment (hagen1778) [inline docs/keyConcepts.md:11]: it is `a metric`, so far. Unless you can explain why `the`

File: docs/keyConcepts.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ It cannot decrease in general case. The only exception is e.g. `counter reset`,
 when the metric resets to zero. The `counter reset` can occur when the service, which exposes the counter, restarts.
 So, the `counter` metric shows the number of observed events since the service start.
 
-In programming, `counter` is a variable that you **increment** each time something happens.
+In programming, the `counter` is a variable that you **increment** each time something happens.
```
Comment (hagen1778) [inline docs/keyConcepts.md:94]: why `the`?

File: docs/keyConcepts.md [hunk 4]
```diff
@@ -108,8 +108,8 @@ The most common [MetricsQL](#metricsql) functions used with counters are:
   time period specified in square brackets.
   For example, `increase(requests_total[1h])` shows the number of requests served over the last hour.
 
-It is OK to have fractional counters. For example, `request_duration_seconds_sum` counter may sum durations of all the requests.
-Every duration may have fractional value in seconds, e.g. `0.5` seconds. So the cumulative sum of all the request durations
+It is OK to have fractional counters. For example, `request_duration_seconds_sum` counter may sum the durations of all the requests.
+Every duration may have a fractional value in seconds, e.g. `0.5` second. So the cumulative sum of all the request durations
```
Comment (hagen1778) [inline docs/keyConcepts.md:112]:
```suggestion
Every duration may have a fractional value in seconds, e.g. `0.5` of a second. So the cumulative sum of all the request durations
```

File: docs/keyConcepts.md [hunk 5]
```diff
@@ -366,9 +366,9 @@ elaborating more on why Percona switched from pull to push model.
 The cons of push protocol:
 
 * Increased configuration complexity for monitored applications.
-  Every application needs te be individually configured with the address of the monitoring system
+  Every application needs to be individually configured with the address of the monitoring system
   for metrics delivery. It also needs to be configured with the interval between metric pushes
-  and the strategy on metric delivery failure.
+  and the strategy for metric delivery failure.
```
Comment (hagen1778) [inline docs/keyConcepts.md:371]:
```suggestion
  and the strategy in case of metric delivery failure.
```

**Review by hagen1778 — CHANGES_REQUESTED** [review]
Thanks! Few comments

File: docs/keyConcepts.md (hunk 2 above)
Comment (YuriKravetc) [reply docs/keyConcepts.md:11]: In both cases, if something was discussed previously (in 1-2 previous sentences it is usually referred to as "the" - though it is uncomfortable to read and might remove it completely.

**Review by hagen1778 — APPROVED** [review]
thanks!

**Pull request merged**